src/components/containers/RecordingsContainer/RecordingsContainer.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+import ArtistList from '../../ArtistList/ArtistList';
+import Loading from '../../Loading/Loading';
+import { useParams } from 'react-router-dom';
+import { getRecordings } from '../../../services/musicBrainz';
+
+export default function RecordingsContainer() {
+  const [results, setResults] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const { id } = useParams();
+
+  useEffect(() => {
+    setLoading(true);
+
+    getRecordings(id)
+      .then(res => {
+        setResults(res.results);
+        setLoading(false);
+      });
+  }, []);
```

Comment:
inside the array, you'll want to put id. There's no hard and fast rule about but if the id changes (and it's in the array) it will trigger a re-render.